### PR TITLE
Start recording ios

### PR DESCRIPTION
**Pull request by rudyZanotti**
Hello

This is just a draft of the video recording for iOS. I know this is far from perfect and could be improved upon, but it works for me and feels like a good start. Hoping someone else can take it back and make it better :)

### CODE REVIEW

File: ios/Plugin/Plugin.swift [hunk 1]
```diff
@@ -246,5 +246,37 @@ public class CameraPreview: CAPPlugin {
...
+                    print(error ?? "Image capture error")
+                    guard let error = error else {
+                        call.reject("Image capture error")
+                        return
+                    }
+                    call.reject(error.localizedDescription)
+                    return
+                }
+
+                    self.videoUrl = image
```
Comment (anth0) [inline ios/Plugin/Plugin.swift:267]: where is this variable declared? I get compilation error in xcode now.

Comment (oneclickwebstudio) [reply ios/Plugin/Plugin.swift:267]: same for me `Value of type 'CameraPreview' has no member 'videoUrl'`

Comment (oneclickwebstudio) [reply ios/Plugin/Plugin.swift:267]: All errors occurring  with Video functions during Swift compile. I thing contributors should roll back or quick fix needed

Comment (facusatoshi) [reply ios/Plugin/Plugin.swift:267]: I'm getting the same error durign swift compile.

Comment (ryaa) [reply ios/Plugin/Plugin.swift:267]: why has this been merged into master? This breaks the build!

Comment (Codemax999) [reply ios/Plugin/Plugin.swift:267]: No hope to get the video items working @rudyZanotti? Would be great, but also see the same errors.

**Pull request merged**